pkg/pillar/types/zedmanagertypes.go

```diff
@@ -128,6 +128,7 @@ type AppInstanceStatus struct {
 	State          SwState
 	MissingNetwork bool // If some Network UUID not found
 	// All error strings across all steps and all StorageStatus
+	RefCount    uint
```

Comment:
What is this RefCount used for? 
Note that the comment above it is about the Error* fields so it is at least misplaced relative to that comment.

Reply:
Please replace with a HasResolverRef boolean in StorageStatus

Reply:
Resolved.